### PR TITLE
fix: three small mapmanip followup issues

### PR DESCRIPTION
**Pull request by warriorstar-orion**
## What Does This PR Do
This PR fixes two small issues after #26803:

- one is rust panic logs not being in gitignore
- one is uploaded map files failing to be read correctly. Mapmanip skips over uploaded map files completely because it can't resolve their path properly. This ensures that the fallback method works as expected. I honestly don't know why the `tfile` var existed, nor why it was constantly being reused, first as the map resource, then its data, but I think this cleanup is safe.
- one is edge marker helpers not qdeling themselves

## Why It's Good For The Game
Admins/coders should be able to upload maps without issue.

## Testing
Ensured server started and loaded all lavaland, ruin, and station DMM files correctly. Used the admin tools to upload an unregistered map template and ensured it was parsed and could place it properly.

<hr>

### Declaration

- [X] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>

## Changelog
NPFC

### CODE REVIEW

File: code/game/objects/effects/map_effects/mapmanip.dm [hunk 1]
```diff
@@ -20,7 +20,15 @@
 	pixel_x = -32
 	pixel_y = -32
 
+/obj/effect/map_effect/marker_helper
+	name = "marker helper"
+	layer = POINT_LAYER
+
+/obj/effect/map_effect/marker_helper/Initialize(mapload)
+	. = ..()
+	qdel(src)
```
Comment (DGamerL) [inline code/game/objects/effects/map_effects/mapmanip.dm:29]: This should do `return INITIALIZE_HINT_QDEL`

**Review by DGamerL — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/map_effects/mapmanip.dm (hunk 1 above)
Comment (warriorstar-orion) [reply code/game/objects/effects/map_effects/mapmanip.dm]: done, PTAL

**Review by DGamerL — APPROVED** [review]
LGTM, waiting on second reviewer

**Review by lewcc — APPROVED** [review]
LGTM

**Pull request merged**